core/peerset/src/lib.rs

```diff
@@ -257,13 +257,18 @@ impl Peerset {
 	}
 
 	fn on_report_peer(&mut self, peer_id: PeerId, score_diff: i32) {
+		// Hotfix: for invulnerable peer connection, check whether the node peer is reserved
+		let reserved = self.data.get_priority_group(RESERVED_NODES).unwrap_or_default();
+		let isReservedPeer = reserved.contains(&peer_id);
```

Comment:
Could u change this to `is_reserved_peer`? (just to be consistent)

Reply:
Nice catch, forgot to use as rust style